train.py

```diff
@@ -29,40 +30,15 @@ def train_single_epoch(epoch, model, train_loader, optimizer, criterion, device,
 
 		total_loss += loss.item()
 
-		print('-'*89)
-		print(f'| epoch {epoch:3d} | {i:5d}/{len(train_loader):5d} batches | loss {total_loss:5.2f} | ppl {math.exp(total_loss):8.2f}')
-		print('-'*89)
-		print(f'Gen: {model.vocab.generate_caption(torch.argmax(output[0].transpose(1,0), dim=-1))}')
-		print(f'Exp: {model.vocab.generate_caption(target[0,1:])}')
-		print('-'*89)
-		candidate_corpus = [model.vocab.generate_caption(torch.argmax(output[0].transpose(1, 0), dim=-1))]
-		reference_corpus = [model.vocab.generate_caption(target[0, 1:])]
-		print(f'Bleu score: {bleu_score(candidate_corpus, reference_corpus)}')
+		if i % 10==0:
```

Comment:
could we remove the `total_loss` variable and just use the `loss.item()` ? It's not related with your changes but since you're modifying this already...

Reply:
Let's discuss it on Friday. I propose to set this as a separate issue to also tackle the log interval and the spaces/tab issue